bin/circos_plot.py

```diff
@@ -0,0 +1,204 @@
+                    r_cds_track.genomic_features([feature], fc="steelblue")
+                if "antismash_bgc_function" in feature.qualifiers:
+                    bgc_track_antismash.genomic_features([feature], fc="tomato")
+                if "gecco_bgc_type" in feature.qualifiers:
+                    bgc_track_gecco.genomic_features([feature], fc="lightsalmon")
+                if "nearest_MiBIG" in feature.qualifiers:
+                    bgc_track_sanntis.genomic_features([feature], fc="firebrick")
+                if "dbcan_prot_type" in feature.qualifiers:
+                    dbcan_track.genomic_features([feature], fc="forestgreen")
+                if "amrfinderplus_scope" in feature.qualifiers:
+                    amr_track.genomic_features([feature], fc="dodgerblue")
+                if "defense_finder_type" in feature.qualifiers:
+                    antiphage_track.genomic_features([feature], fc="orchid")
+
+            elif feature.type in ["tRNA", "ncRNA"]:
+                rna_track.genomic_features([feature], fc="darkmagenta")
+            elif mobilome and feature.type in [
+                "mobility_island",
+                "cellular_recombinase",
+                "insertion_sequence",
+                "conjugative_element",
+                "nested_mobile_element",
+                "terminal_inverted_repeat_element",
+                "direct_repeat_element",
+                "viral_sequence",
+            ]:
+                mobilome_track.genomic_features([feature], fc="lightseagreen")
+            elif mobilome and feature.type.lower() == "phage":
+                mobilome_track.genomic_features([feature], fc="blue")
+
+    fig = circos.plotfig()
+    # Add legend
+    handles = [
+        Patch(color="hotpink", label="Forward CDS"),
+        Patch(color="steelblue", label="Reverse CDS"),
+        Patch(color="darkmagenta", label="RNA"),
+        Patch(color="tomato", label="BGCs (antiSMASH)"),
+        Patch(color="lightsalmon", label="BGCs (GECCO)"),
+        Patch(color="firebrick", label="BGCs (SanntiS)"),
+        Patch(color="forestgreen", label="Predicted PULs"),
+        Patch(color="dodgerblue", label="AMR genes"),
+        Patch(color="orchid", label="Anti-phage defense genes"),
+    ]
+    if mobilome:
+        handles = handles + [
+            Patch(color="blue", label="Mobilome (phage)"),
+            Patch(color="lightseagreen", label="Mobilome (other)"),
+        ]
+
+    main_legend = circos.ax.legend(
+        handles=handles, bbox_to_anchor=(0.5, 0.475), loc="center", fontsize=8
+    )
+    circos.ax.add_artist(main_legend)
+    fig.savefig(outfile, dpi=600)
+
+
+def remove_escaped_characters(infile):
+    outfile = infile + "_modified"
+    with open(infile, "r") as file_in:
+        content = file_in.read()
+        modified_content = content.replace("\\=", "")
+
+    # Open the file in write mode to overwrite it with the modified content
+    with open(outfile, "w") as file_out:
```

Comment:
I would make a copy, re-writing files is not what most users would expect

Reply:
It's a poorly worded comment on my part, it is making a copy. I will edit it for clarity.

Reply:
ahh.. my bad. I misread the code